lib/inputs/byte-downloads.js

```diff
@@ -38,64 +28,81 @@ module.exports = class ByteDownloads {
       return Promise.resolve([]);
     } else {
       const originals = await this.lookup();
-      const filtered = this.filter(originals);
+
+      // format using the original ddb record
+      const formatted = this._records.map(r => {
```

Comment:
The logic here simplifies quite a bit, now that we're outputting one kinesis record per bytes/segmentbytes input.

(The exception being if no DDB record was found).